control/metrics.go

```diff
@@ -625,6 +625,9 @@ func addStandardAndWorkflowTags(m core.Metric, allTags map[string]map[string]str
 	if tags == nil {
 		tags = map[string]string{}
 	}
+	// apply standard tag
+	tags[core.STD_TAG_PLUGIN_RUNNING_ON] = hostname
+
```

Comment:
To be clear this is done to ensure that the tags from the workflow can overwrite this if they want to?